api-datos/cmd/tablacruce/main.go

```diff
@@ -0,0 +1,45 @@
+// Este archivo contiene la función main
+package main // import "bitbucket.org/IslasGECI/tablacruce"
+
+import (
+	"database/sql"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+type App struct {
+	*sql.DB
+	*mux.Router
+}
+
+func RequestLogger(targetMux http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		targetMux.ServeHTTP(w, r)
+		requesterIP := r.RemoteAddr
+		log.Printf(
+			"%s - - \"%s %s\"",
+			requesterIP,
+			r.Method,
+			r.RequestURI,
```

Comment:
¿Podrás quitar la coma?
```suggestion
			r.RequestURI
```

Reply:
No sé puede, eso es parte del lenguaje